templates/zerver/help/keyboard-shortcuts.md

```diff
@@ -69,6 +69,8 @@ Zulip keyboard shortcuts are divided into four categories:
   messages with the topic of the selected message (outlined in blue).
 * **Narrow to all private messages**: `P` - This shortcut narrows the
   view to show all of the user's private messages.
+* **Narrow to next unread topic**: `P` - This shortcut narrows the
+  view to the next unread topic in the stream sidebar.
```

Comment:
Note to @timabbott: We may eventually want to document the nuances of this--for example, if you're at the bottom of the streams, it will loop back to the top--but for now I think simpler is better.